src/views/components/student-panel.jsx

```diff
@@ -59,69 +60,76 @@ export default class StudentPanel extends ListenerComponent {
+                                <button id={model.address} onClick={() => this.checkInModel(this.props.student.id, model.address, parseInt(this.state.checkinNum))}>Check in</button>
+                                <button id={`all${model.address}`}  onClick={() => this.checkInModel(this.props.student.id, model.address, model.quantity)}>Check in All</button>
+                                <button onClick={() => StudentPanelController.reserveModel(this.props.student.id, model.address)}>Save</button>
+                            </div>
                         </div>
                     );
                 })}
             </div>
         );
     }
 
-    renderModelInfo(model){
-        return (<div>{model.name} <i>{model.address}</i> ({model.quantity})</div>);
-    }
-
-    renderCheckinButtons(model){
+    renderModelInfo(model) {
         return (
-            <div className='checkin-buttons'>
-                <input type='number'  value={this.state.checkinNum} onChange={this.changeCheckinNum.bind(this, model.quantity)} min='1' max={model.quantity} />
-                <button id={model.address} onClick={() => this.checkInModel(this.props.student.id, model.address, parseInt(this.state.checkinNum))}>Check in</button>
-                <button id={`all${model.address}`}  onClick={() => this.checkInModel(this.props.student.id, model.address, model.quantity)}>Check in All</button>
-            </div>
+            <span>
+                {model.name} <i>{model.address}</i> ({model.quantity})
+            </span>
         );
     }
 
-    renderItemInfo(item){
+    renderItemInfo(item) {
         let model = this.state.models.find(model => model.address === item.modelAddress);
-        if(!model){
+        if (!model) {
             return null;
-        }else{
+        } else {
             let dueDate = moment.tz(item.timestamp * 1000, 'America/Chicago');
-            return (<div>
-                {model.name} {item.timestamp < Math.floor(Date.now()/1000) ? '(overdue)' : ''} <i>{item.address} Due on: {dueDate.format('MMMM Do YYYY, h:mm:ss a')}</i>
-            </div>)
+            return (
+                <div>
+                    {model.name} {item.timestamp < Math.floor(Date.now()/1000) ? '(overdue)' : ''} <i>{item.address} Due on: {dueDate.format('MMMM Do YYYY, h:mm:ss a')}</i>
+                </div>
+            );
         }
     }
 
     render() {
+        let savedItems = this.props.student.items.filter(item => item.status === 'SAVED');
+        let savedModels = this.props.student.models.filter(model => model.status === 'SAVED');
         return (
             <div className='student'>
                 <h2 className='name'>{this.props.student.name}</h2>
                 <i className='id'>{this.props.student.id}</i>
                 <h4 className='equipment-heading'>Equipment</h4>
                 {this.renderEquipment()}
+                <SavedEquipment items={savedItems} models={savedModels} />
```

Comment:
Nit: To more a bit more consistent, the \<h4\> should be kept outside \<SavedEquipment\> so that only the "equipment bars" are in \<SavedEquipment\>.